client/src/pages/Search.jsx

```diff
@@ -1,27 +1,14 @@
 import CardTrack from '../components/CardTrack'
```

Comment:
Try to follow the "best practice" example from Thomas today for imports :-) 

– Import der Systemkomponenten (useState, useEffect, react-router-dom, styled-components etc.)
– Die eigenen React-Komponenten
– Libraries (validate.js, fetchApi, calculator, favourites, localStorage etc.)
– Styling / CSS files
– Icons / Bilder